src/Strings.jl

```diff
@@ -1,9 +1,98 @@
 module Strings
 
-export escapehtml, tocameldash, iso8859_1_to_utf8, ascii_lc_isequal
+export HTTPVersion, escapehtml, tocameldash, iso8859_1_to_utf8, ascii_lc_isequal
```

Comment:
this is in `src/Strings.jl` because it was the submodule that was imported in exactly the same places we need  `HTTPVersion` 

happy to move it somewhere else

Reply:
I think that's fine